lib/forms/CoBrALab-Mouse-Surgery-Form/index.ts

```diff
@@ -0,0 +1,304 @@
+          }
+        }
+        return null
+      }
+      
+    },
+    stereotaxUsed: createDependentField({
+      kind: "boolean",
+      variant: "radio",
+      label: "Stereotax used"
+    },(type) => type === "Surgery"),
+
+    stereotaxId:{
+      kind: "dynamic",
+      deps: ["stereotaxUsed"],
+      render(data) {
+        if(data.stereotaxUsed){
+          return {
+            kind: "string",
+            variant: "input",
+            label: "Stereotax ID"
+          }
+        }
+        return null
+      }
+
+    },
+
+    surgeryType: createDependentField({
+      kind: "string",
+      variant: "select",
+      label: "Type of surgery",
+      options: {
+        "Ovariectomy": "Ovariectomy",
+        "Electrode implant": "Electrode implant",
+        "Fiber optic implant": "Fiber optic implant"
+      }
+    },(type) => type === "Surgery"),
+    ovariectomyType: {
+      kind: "dynamic",
+      deps: ["surgeryType"],
+      render(data) {
+        if(data.surgeryType === "Ovariectomy"){
+          return {
+             kind: "string",
+             variant: "select",
+             label: "Ovariectomy sides",
+             options: {
+             "Unilateral":"Unilateral",
+             "Bilateral": "Bilateral"
+             }
+          }
+        }
+        return null
+      }
+    },
+    ovariectomySide: {
+      kind: "dynamic",
+      deps: ["ovariectomyType"],
+      render(data) {
+        if(data.ovariectomyType === "Unilateral"){
+          return {
+             kind: "string",
+             variant: "select",
+             label: "Ovariectomy side",
+             options: {
+             "Right":"Right",
+             "Left": "Left"
+             }
+          }
+        }
+        return null
+      }
+    },
+    ovariectomyMouseGroup: {
+      kind: "dynamic",
+      deps: ["surgeryType"],
+      render(data) {
+        if(data.surgeryType === "Ovariectomy"){
+          return {
+             kind: "string",
+             variant: "select",
+             label: "Ovariectomy group (control/experiment)",
+             options: {
+             "Control":"Control",
+             "Experiment": "Experiment"
+             }
+          }
+        }
+        return null
+      }
+    },
+
+    brainSurgeryLocation: createSurgeryDependentField({
+      kind: "string",
+      variant: "select",
+      label: "Brain surgery location",
+      options: {
+        "Bregma": "Bregma",
+        "Paxinos coordinates": "Paxinos coordinates"
+      }
+    },(type) => type === "Electrode implant" || type === "Fiber optic implant"),
+
+    brainSurgeryPaxinosCoords: {
+      kind: "dynamic",
+      deps: ["stereotaxUsed","brainSurgeryLocation"],
+      render(data) {
+        if(data.brainSurgeryLocation === "Paxinos coordinates" && data.stereotaxUsed){
+          return {
+            kind: "string",
+            variant: "input",
+            label: "Paxinos coordinates for surgery"
+          }
```

Comment:
Please discuss with the users a better way to document this.

This should be a proper structured input, probably multiple fields

Feel free to expand the description here to ensure they get it right.

Reply:
I can base this questions of this paxinos coordinate calculator [here](https://labs.gaidi.ca/mouse-brain-atlas/) and I can set the limit based in what i see in the diagram.